### PR TITLE
New modal

### PR DESCRIPTION
**Pull request by raphaeldieu**
Use ui-bootsrap

### CODE REVIEW

File: app/assets/javascripts/angular/controller/forms/listForms.js.erb [hunk 1]
```diff
@@ -42,20 +26,23 @@ app.controller('ListFormsCtrl', ['$scope', '$http','$auth','$location','$rootSco
...
+      animation: $scope.animationsEnabled,
+      templateUrl: "<%= asset_path('forms/modalCreateForms.html') %>",
+      controller: 'ModalInstanceCtrl'
     });
+
   };
 
-  $scope.close = function() {
-    close(500); 
+  $scope.toggleAnimation = function () {
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms/listForms.js.erb]: Do you understand what this function does? Why do you want to allow the user to choose whether or not there will be animations for modals?

File: app/assets/templates/forms/modalCreateForms.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="modal-header">
+    <h5 class="modal-title">Give your form a name</h5>
+</div>
+<div class="modal-body">
+  <input type="text" class="" placeholder="e.g. Market research" ng-model="form.name" style="height: 50px;">
+</div>
+<div class="modal-footer">
+    <button class="btn btn-primary" type="button" ng-click="ok()">Start to buid your new form!</button>
+    <button class="btn btn-warning" type="button" ng-click="cancel()">Cancel</button>
```
Comment (h8rry) [inline app/assets/templates/forms/modalCreateForms.html]: Indentations are a little off?

**Comment by h8rry** [comment]
@raphaeldieu please review

**Comment by raphaeldieu** [comment]
@ggharry 
should be better now :)

**Pull request merged**